MapboxCoreNavigation/Constants.swift

```diff
@@ -110,81 +110,15 @@ public var MaxSecondsSpentTravelingAwayFromStartOfRoute: TimeInterval = 3
 
 
 /**
- Distance in meters for the minimum length of a step for giving a `medium` alert while using `MBDirectionsProfileIdentifierAutomobile` or `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`.
+ Remaing distance on a motorway at which the `.high` `AlertLevel` will be given. This overrides `RouteControllerHighAlertInterval` only when the current step is a motorway. Default value is a half mile.
  */
-public var RouteControllerMinimumDistanceForMediumAlertDriving: CLLocationDistance = 400
+public var RouteControllerMotorwayHighAlertDistance: CLLocationDistance = 804.672
```

Comment:
It isn’t apparent from reading `804.672` that this variable is set to the right default value. To make it apparent, set it to `0.5 * milesToMeters` instead. (×2)